doc/source/structures/vessels/crewmember.rst

```diff
@@ -0,0 +1,78 @@
+.. _crewmember:
+
+CrewMember
+===========
+
+Represents a single crew member of a vessel.
+
+.. structure:: CrewMember
+
+    .. list-table::
+        :header-rows: 1
+        :widths: 2 1 4
+
+        * - Suffix
+          - Type
+          - Description
+
+        * - :attr:`NAME`
+          - string
+          - crew member's name
+        * - :attr:`GENDER`
+          - string
+          - "Male" or "Female"
+        * - :attr:`EXPERIENCE`
+          - scalar
+          - experience level
+        * - :attr:`TRAIT`
+          - string
+          - "Pilot", "Engineer" or "Scientist"
+        * - :attr:`TOURIST`
+          - Boolean
+          - true if this crew member is a tourist
+        * - :attr:`PART`
+          - :struct:`Part`
+          - part in which the crew member is located
+
+
+.. attribute:: CrewMember:NAME
+
+    :type: string
+    :access: Get only
+
+    crew member's name
+
+.. attribute:: CrewMember:GENDER
+
+    :type: string
+    :access: Get only
+
+    "Male" or "Female"
+
+.. attribute:: CrewMember:EXPERIENCE
+
+    :type: scalar
+    :access: Get only
+
+    experience level
```

Comment:
Is this the number of stars, or the more fine-grain experience level, as in it could tell you how far along the slider to the next star you are?  The documentation should probably make it clear which it is.